internal-packages/next-dev/README.md

```diff
@@ -54,7 +54,8 @@ export default nextConfig;
 // (note: this check is recommended but completely optional)
 if (process.env.NODE_ENV === 'development') {
 	// we simply need to call the utility
-	setupDevPlatform();
+	// (`await`ing the call is not mandatory but recommended)
```

Comment:
Note: this comment might look strange, I wanted to add it because top level awaits are not allowed in cjs files, so if someone were to have a `next.config.(js/cjs)` then they could not `await` the function call, so hopefully this can avoid some confusion (without being too verbose/descriptive)

please let me know what you think 🙂

Reply:
We should add why we recommend it to make this fully complete. It's ok to be verbose on the comments as it's supposed to be documentation anyway. 

Something like:

> The await only helps verify that the setup has succeeded, and is not mandatory. If using a .js/cjs config format that you can safely remove this top level await.

Reply:
cool, thanks 🙂 

I've updated the comment please let me know what you think